apt-bootstrap

```diff
@@ -375,6 +375,7 @@ class AptBootstrap(object):
         self.mount('-t', 'proc', 'proc', os.path.join(self.path, 'proc'))
         self.mount('-t', 'sysfs', 'sysfs', os.path.join(self.path, 'sys'))
         self.mount('--bind', '/dev/pts', os.path.join(self.path, 'dev/pts'))
+        self.mount('--bind', '/tmp', os.path.join(self.path, 'tmp'))
```

Comment:
Interesting. This must be new dpkg behavior. I don't recall hitting anything like this 2 years ago.